tests/zenko_e2e/backbeat/tests/crr-pause-resume/awsBackend.js

```diff
@@ -0,0 +1,107 @@
+const assert = require('assert');
+const crypto = require('crypto');
+const  { series, parallel, timesSeries, each } = require('async');
+
+const { scalityS3Client, awsS3Client } = require('../../s3SDK');
+const ReplicationUtility = require('../../ReplicationUtility');
+
+const scalityUtils = new ReplicationUtility(scalityS3Client);
+const awsUtils = new ReplicationUtility(awsS3Client);
+const srcBucket = `source-bucket-${Date.now()}`;
+const destBucket = process.env.AWS_S3_BACKBEAT_BUCKET_NAME;
+const destLocation = process.env.AWS_DESTINATION_LOCATION;
+const hex = crypto.createHash('md5')
+    .update(Math.random().toString())
+    .digest('hex');
+const keyPrefix = `${srcBucket}/${hex}`;
+const key = `${keyPrefix}/object-to-replicate-${Date.now()}`;
+const copyKey = `${key}-copy`;
+// eslint-disable-next-line
+const keyutf8 = `${keyPrefix}/%EA%9D%8B崰㈌㒈保轖䳷䀰⺩ቆ楪僷ꈅꓜ퇬枅࿷염곞召㸾⌙ꪊᆐ庍뉆䌗↎幐냂詴 끴鹲萯⇂쫤ᛩ꺶㖭簹릍铰᫫暨鿐魪셑蛃춧㡡竺뫁噛̷ᗰⷑ錜⑔痴䧫㾵᏷ำꎆ꼵껪멷㄀誕㳓腜쒃컹㑻鳃삚舿췈孨੦⮀Ǌ곓⵪꺼꜈嗼뫘悕錸瑺⁤⑬১㵀⡸Ҏ礄䧛졼⮦ٞ쫁퓡厈譤擵泶鵇俻縫륒㜓垻ㆩꝿ詀펉ᆙ舑䜾힑藪碙ꀎꂰ췊Ᏻ   㘺幽醛잯ද汧Ꟑꛒⶨ쪸숞헹㭔ꡔᘼ뺓ᡆ᡾ᑟ䅅퀭耓弧⢠⇙폪ް蛧⃪Ἔ돫ꕢ븥ヲ캂䝄쟐颺ᓾ둾Ұ껗礞ᾰ瘹蒯硳풛瞋襎奺熝妒컚쉴⿂㽝㝳駵鈚䄖戭䌸᫲ᇁ䙪鸮ᐴ稫ⶭ뀟ھ⦿䴳稉ꉕ捈袿놾띐✯伤䃫⸧ꠏ瘌틳藔ˋ㫣敀䔩㭘식↴⧵佶痊牌ꪌ搒꾛æᤈべ쉴挜炩⽍舘ꆗ줣徭Z䐨 敗羥誜嘳ֶꫜ걵ࣀ묟ኋ拃秷䨸菥䟆곘縧멀煣⧃⏶혣뎧邕⢄⭖陙䣎灏ꗛ僚䌁䠒䲎둘ꪎ傩쿌ᨌ뀻阥눉넠猌ㆯ㰢船戦跏灳蝒礯鞰諾벥煸珬㟑孫鞹Ƭꄹ孙ꢱ钐삺ᓧ鈠䁞〯蘼᫩헸ῖ"`;
+const REPLICATION_TIMEOUT = 300000;
+
+describe('Replication Pause-Resume with AWS backend', function() {
+    this.timeout(REPLICATION_TIMEOUT);
+    let roleArn = 'arn:aws:iam::root:role/s3-replication-role';
+
+    beforeEach(done => series([
+        next => scalityUtils.createVersionedBucket(srcBucket, next),
+        next => scalityUtils.putBucketReplicationMultipleBackend(srcBucket,
+            destBucket, roleArn, destLocation, next),
+    ], done));
+
+    afterEach(done => series([
+        next => scalityUtils.deleteVersionedBucket(srcBucket, next),
+        next => awsUtils.deleteAllVersions(destBucket,
+            `${srcBucket}/${keyPrefix}`, next),
+    ], done));
+
+    after(done => scalityUtils.resumeReplication(null, null, null, done));
+
+    it('should pause and resume replication', done => series([
+        next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(1), next),
+        next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
```

Comment:
I think we can just wait for replication to complete—i.e. no need to get and compare the objects since that is being tested elsewhere. There is a helper method for this `waitUntilReplicated`. Probably the same applies for tests below that are comparing the objects.